Week1/homework/js-exercises/exercise5.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+const z = 7.25;
+console.log(z);
+const a = 7;
+console.log(a);
+const biggistNumber = a > z ? a : z;
```

Comment:
Ah nice, you can also use [Math.max](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max)!